bittensor/commands/wallets.py

```diff
@@ -73,11 +73,11 @@ async def run(cli: "bittensor.cli"):
             # Password can be "", assume if None
             json_password = cli.config.get("json_password", "")
         wallet.regenerate_coldkey(
+            use_password=cli.config.use_password,
+            overwrite=cli.config.overwrite_coldkey,
```

Comment:
Why are these changed? Again, it's non-blocking. Just looking for rationale

Reply:
In the function `wallet.regenerate_coldkey`, `use_password` and `overwrite` are two first arguments. 
This is making readability more obvious and shows us that the last arguments passed as a part of `**kwargs`